scripts/ci/batch_pw_load.sh

```
@@ -0,0 +1,325 @@
+    fi
+
+    echo "Copying TypeScript configuration files..."
+    docker compose -f "$COMMON_HEALTHCHECKS_FILE" -f "$DOCKER_COMPOSE_TEST_FILE" cp tsconfig.json playwright:/app/ || echo "⚠️  Failed to copy tsconfig.json"
+    docker compose -f "$COMMON_HEALTHCHECKS_FILE" -f "$DOCKER_COMPOSE_TEST_FILE" cp tsconfig.paths.json playwright:/app/ || echo "⚠️  Failed to copy tsconfig.paths.json"
+    docker compose -f "$COMMON_HEALTHCHECKS_FILE" -f "$DOCKER_COMPOSE_TEST_FILE" cp next.config.js playwright:/app/ || echo "⚠️  Failed to copy next.config.js"
+    docker compose -f "$COMMON_HEALTHCHECKS_FILE" -f "$DOCKER_COMPOSE_TEST_FILE" cp playwright.config.ts playwright:/app/ || echo "⚠️  Failed to copy playwright.config.ts"
+
+    echo "🔍 Verifying files were copied correctly..."
+    docker compose -f "$COMMON_HEALTHCHECKS_FILE" -f "$DOCKER_COMPOSE_TEST_FILE" exec -T playwright ls -la /app/src/test/visual/ || echo "⚠️  Visual files not found in container"
+    
+    run_visual_tests_dind() {
```

Shellcheck (0.10.0)

[info] The mentioned syntax error was in this function.

(SC1009)